ProcessLib/BoundaryCondition/BHEBottomDirichletBoundaryCondition.cpp

```diff
@@ -0,0 +1,78 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "BHEBottomDirichletBoundaryCondition.h"
+
+#include <algorithm>
+#include <logog/include/logog.hpp>
+#include <vector>
+#include "ProcessLib/Utils/ProcessUtils.h"
+
+namespace ProcessLib
+{
+void BHEBottomDirichletBoundaryCondition::getEssentialBCValues(
+    const double /*t*/, GlobalVector const& x,
+    NumLib::IndexValueVector<GlobalIndexType>& bc_values) const
+{
+    SpatialPosition pos;
+
+    bc_values.ids.clear();
+    bc_values.values.clear();
```

Comment:
Is the clear call needed? It looks like you set all the ids and values in the for-loop below anyway.

Reply:
They are not needed. Removed.